packages/ngtools/webpack/src/virtual_file_system_decorator.ts

```diff
@@ -18,17 +18,16 @@ export class VirtualFileSystemDecorator implements InputFileSystem {
     private _webpackCompilerHost: WebpackCompilerHost,
   ) { }
 
-  // We only need to intercept calls to individual files that are present in WebpackCompilerHost.
   private _readFileSync(path: string): string | null {
```

Comment:
Webpack filesystems return a Buffer not a string.  This will break any file type that is not text (e.g., JPG, WOFF, etc.)  or uses non-default text encoding (i.e., not UTF8 without a BOM).

Reply:
I agree. I'll keep digging into that. Do you have some ideas we might try?

Reply:
Yes.  Unfortunately, it is a decent amount of work.  The `WebpackCompilerHost` would need to be refactored to internally use `Buffer` objects to store the data and then convert to strings where needed inside the `WebpackCompilerHost`.  And then provide `Buffer` return objects for the read calls within the webpack filesystem decorators.